### PR TITLE
implement activity logs changes

### PR DESCRIPTION
**Pull request by bashaashma**
# Description

Added additional changes to activity logs

## Type of change

Please delete options that are not relevant.

- [X] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)

## Trello Ticket ID

https://trello.com/c/TKwkNJ4p/1311-user-activity-logs-changes

## How Can This Be Tested?

Input project id, app id or database id for activity logs. User should be able to see all activities for that project, app or database done by other users as well. Non-member users cannot see logs for a project. The logs should now also include the user email and user name.

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] My changes generate no new warnings
- [X] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by MutegekiHenry — DISMISSED** [review]
Works fine here

File: app/controllers/users.py [hunk 1]
```diff
@@ -786,11 +788,46 @@ def get(self):
...
 
-            # check for start and end query params
+            # check if owner or admin or member for project, database or app
+            project_id = None
+            if validated_data_query.get('a_project_id'):
+                project_id = validated_data_query.get('a_project_id')
+                validated_data_query.pop('user_id',None)
+                
+            elif validated_data_query.get('a_db_id'):
+                validated_data_query.pop('user_id',None)
```
Comment (Mubangizi) [inline app/controllers/users.py:802]: I get the reason for removing the `user_id` from the filters. I guess that's to be able to get logs for all users in a **Project**.
There is a scenarial we are not handling and that is if I want to see only activity for a particular user in a project. Meaning I would have to pass `user_id` that is not mine.

Hence you might need to add the case for:

` If validated_data['user_id'] and user_id != current_user_id:
    // You let the user_id reach the db as query
`

**Review by Mubangizi — COMMENTED** [review]
Hey @bashaashma, this is great work well done. There is a certain small issue that is missing for this to be complete.

**Review by Mubangizi — APPROVED** [review]
Great work @bashaashma.

**Review by e-ian — APPROVED** [review]
LGTM

**Pull request merged**